src/Product/ProductList.js

```diff
@@ -0,0 +1,65 @@
+import { useState, useEffect } from "react";
+import Product from "./Product";
+
+const API_URL = "https://px32id5fdg.execute-api.us-east-1.amazonaws.com/data";
+
+function ProductList() {
+  const [productData, setProductData] = useState([]);
+  const [searchInput, setSearchInput] = useState("");
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    fetch(API_URL)
+      .then((response) => {
+        return response.json();
+      })
+      .then((data) => {
+        setProductData(data);
+        setLoading(false);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }, []);
+
+  const filteredProductData = productData
+    .filter((product) => {
+      return (
+        product.name.toLowerCase().includes(searchInput.toLocaleLowerCase()) ||
+        product.shortDescription
+          .toLowerCase()
+          .includes(searchInput.toLocaleLowerCase())
+      );
+    })
+    .map((product) => {
+      return <Product product={product} key={product.id} />;
```

Comment:
This is a nice approach to chain filtering and mapping to JSX elements. You could take this a step further and use `.reduce()` so that you get an array of `<Product />` in a single pass through `productData`. (Not required, just something to try if you are curious.)